src/components/claim/ClaimRewardsVeDF.svelte

```diff
@@ -146,11 +148,24 @@
     streams[1].substreams[1].availableRewards = challengeRewards
   }
 
+  const calculateUnclaimedPassiveReward = () => {
+    let sum = 0
+      $oceanUserRewards.forEach((r) => {
+        if(r.round >= $lastPassiveRewardsClaimRound) {
+          sum += r['sum(passive_amt)']
+        }
+      })
+      passiveRewards = sum
+      streams[0].substreams[0].availableRewards = sum
+  }
+
+  $:if($lastActiveRewardsClaimRound >= 0 && $oceanUserRewards) calculateUnclaimedPassiveReward()
```

Comment:
Hi, just to be clear, it seems like the logic is now aggregating all claimable rewards.

This is different than what I heard in the stand up, where only the next one (rather than multiples) will be visible to the user.

If all rewards are displayed and user can start claiming them, then great!

Reply:
I left some comments here:
https://github.com/oceanprotocol/df-web/issues/637#issuecomment-1765325539

I think what you mean is "Next Claim {amount}" rather than "First Claim {amount}".

If correct, please adjust the button text to be more intuitive.

Reply:
I updated based on the discussion from within the issue